lib/rubygems/gemcutter_utilities.rb

```diff
@@ -267,6 +269,27 @@ def get_scope_params(scope)
     scope_params
   end
 
+  def get_mfa_params(email, password)
+    mfa_level = get_user_mfa_level(email, password)
+    params = {}
+    if mfa_level == "ui_only" || mfa_level == "ui_and_gem_sign"
```

Comment:
```suggestion
    if mfa_level == "ui_only" || mfa_level == "ui_and_gem_signin"
```
I think it's [`ui_and_gem_signin`](https://github.com/rubygems/rubygems.org/blob/master/app/models/user.rb#L61)

Reply:
Doh! Updated and adjusted the test so a future mistake like this will cause a failure